portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,49 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Chelsea+Market&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <div id="header">
+        <div id="home" class="headBox"> <a href="index.html"> Home </a> </div>
+    </div>
+
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Erin's Portfolio</h1>
+        <p>This is my portfolio.</p>
+        <p>Click here to see yours truly ;)</p>
+        <button onclick="showPicture()">See me!</button>
+        <div id="content-container">
+            <div> <img id="portrait" src="images/portrait.JPG" height="20px"/> </div>
+            <div id="bio-container">
+                <div class="bio"> 
```

Comment:
same thing here, since bio is only defined once, you may want to use `id` instead